Composer/package.json

```diff
@@ -15,7 +15,7 @@
     "serialize-javascript": "^3.1.0",
     "set-value": "^3.0.2",
     "terser-webpack-plugin": "^2.3.7",
-    "axios": "^0.21.1",
+    "axios": "^0.22.0",
```

Comment:
This needs either to include updated lock files or exluding the change from the PR